src/uniref/components/results/UniRefCard.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { FC } from 'react';
+import { Card } from 'franklin-sites';
+import { useHistory } from 'react-router-dom';
+
+// import { UniProtkbAPIModel } from '../../adapters/uniProtkbConverter';
```

Comment:
Should all of the commented code in this file stay for now or it can be added later on, if needed?

Reply:
I'd keep it for now, to clearly show that this file is a fork of the UniProtKB version of it. Whenever it's clearer what needs to go in there it might make it more obvious that there is similar code in UniProtKB because it's here in the comments and we'd need to generalise it. Otherwise we can remove it later when we flesh out this component.
In any case, we are going to have to go over this file in the near future.